app/scripts/lib/user-agent.js

```diff
@@ -31,6 +31,15 @@ define(function (require, exports, module) {
       },
 
       /**
+       * Check if on a desktop device
+       *
+       * @returns {Boolean}
+       */
+      isDesktop () {
+        return ! this.isAndroid() && ! this.isIos();
```

Comment:
Although it's not significant for the correctness of this PR, going forward I think this is a mildly surprising definition for an `isDesktop` predicate. It seems like what we're really expressing is something more like `isPlatformThatDoesntHaveAnAppStore` (thank you, unwieldy names are my forté).

I wouldn't have bothered mentioning it, but the fact that this lives in a generic user-agent module makes me more cautious. Might this be a name we regret using in the future? Is there a better name we could come up with? (and not my ridiuclous suggestion from the previous paragraph)

Or maybe we should just suck it up and force callers to inline this function themselves? Explicitly writing:

```
! this.isAndroid() && ! this.isIos()
``` 

...doesn't seem lilke a huge imposition and is more honest than `isDesktop` in this case.